main.js

```diff
@@ -175,7 +175,7 @@ function getWeatherCompanyForecast(geocode) {
         const result = results[i];
         console.log(result);
         console.log(result.name);
-        if (result.name.startsWith('Bluemix_Weather Company Data')) {
+        if (result.name.startsWith('IBM Cloud_Weather Company Data')) {
```

Comment:
this potentially breaks code.  Needs to be verified.  I'm assuming this was just a search and replace and should b e rejected.